tc39-notes/2012-05_may-21.html

```diff
@@ -0,0 +1,3 @@
+<!doctype html>
+<link rel="canonical" href="https://github.com/tc39/notes/blob/master/meetings/2012-05/may-21.md">
+<meta http-equiv=refresh content="0;URL=https://github.com/tc39/notes/blob/master/meetings/2012-05/may-21.md">
```

Comment:
can you add a trailing newline to all these files?

Reply:
newline added